src/ui/routes/+page.svelte

```diff
@@ -1,3 +1,63 @@
-<section>
-	<h3>Zarf UI landing page</h3>
-</section>
+<script>
+	import { Button } from '@ui';
+	import bigZarf from '@images/zarf-bubbles.png';
+	let zarfStatus = false;
+</script>
+
+<svelte:head>
+	<title>Zarf</title>
+</svelte:head>
+
+{#if zarfStatus === false}
+	<section class="hero">
```

Comment:
Isn't hero a bootstrap term for this?  Am I crazy?

Reply:
ya, `hero` is the classic term for a full centered page, usually a landing page with a "call to action" portion.  This case being the detection that zarf init has not run.